flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
+    "x86_64"
```

Comment:
this is disappointing.

Reply:
Well I can have a look to add aarch64 build in the CI

Reply:
Don't have anything to make arm64 build at the moment.